src/components/admin/ImageController.jsx

```diff
@@ -138,6 +148,14 @@ export default class ImageUploader extends BaseComponent {
       this.addImagePreviewUrl(file);
     });
 
+    if (tooLargeFileNames.length !== 0) {
+      let names = '\n';
+      tooLargeFileNames.forEach((file) => {
+        names = names.concat(` ${file.name} (${(file.size / 1024).toFixed(2)} KB)\n`);
+      });
+      alert(`The following exceeded the maximum upload size: ${names}`);
+    }
+
     // Reset to "No Files Chosen" in the input element instead of saying "10 files" or so
     e.target.parentNode.parentNode.parentNode.parentNode.parentNode.reset();
```

Comment:
There must be a more elegant way to do this. What are all the `.parentNode`s referring to here??

Reply:
If I remember correctly, it was just really hard getting a ref to it or something like that, it is without a doubt ugly though